src/App.js

```diff
@@ -75,72 +74,37 @@ function App() {
     setPlayer("O");
   };
 
-  return (
-    <div className="App">
-      <div className="board">
-        <div className="row">
-          <Square
-            val={board[0]}
-            chooseSquare={() => {
-              chooseSquare(0);
-            }}
-          />
-          <Square
-            val={board[1]}
-            chooseSquare={() => {
-              chooseSquare(1);
-            }}
-          />
-          <Square
-            val={board[2]}
-            chooseSquare={() => {
-              chooseSquare(2);
-            }}
-          />
-        </div>
-        <div className="row">
-          <Square
-            val={board[3]}
-            chooseSquare={() => {
-              chooseSquare(3);
-            }}
-          />
-          <Square
-            val={board[4]}
-            chooseSquare={() => {
-              chooseSquare(4);
-            }}
-          />
-          <Square
-            val={board[5]}
-            chooseSquare={() => {
-              chooseSquare(5);
-            }}
-          />
-        </div>
-        <div className="row">
-          <Square
-            val={board[6]}
-            chooseSquare={() => {
-              chooseSquare(6);
-            }}
-          />
-          <Square
-            val={board[7]}
-            chooseSquare={() => {
-              chooseSquare(7);
-            }}
-          />
+    // Use two loops to make the squares
+    const boardSize = 3;
+    let squares = [];
+    for (let i = 0; i < boardSize; ++i) {
+      let row = [];
+      for (let j = 0; j < boardSize; ++j) {
+        row.push(
           <Square
-            val={board[8]}
+            val={board[i * boardSize + j]}
             chooseSquare={() => {
-              chooseSquare(8);
+              chooseSquare(i * boardSize + j);
             }}
           />
+        );
+       // squares.push(<div className="row"></div>); remove key;
```

Comment:
Usually you would remove dead comments like this once you're done with them.

Reply:
Thanks for the reminder!